src/browser/components/UpdaterPage/UpdaterPage.stories.jsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-2016 Yuya Ochiai
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import {action} from '@storybook/addon-actions';
+
+import UpdaterPage from '../UpdaterPage.jsx';
+import '../../css/components/UpdaterPage.css';
+
+/*
+appName: propTypes.string.isRequired,
+notifyOnly: propTypes.bool.isRequired,
+isDownloading: propTypes.bool.isRequired,
+progress: propTypes.number,
+onClickInstall: propTypes.func.isRequired,
+onClickDownload: propTypes.func.isRequired,
+onClickReleaseNotes: propTypes.func.isRequired,
+onClickRemind: propTypes.func.isRequired,
+onClickSkip: propTypes.func.isRequired,
+*/
+const appName = 'Storybook App';
+
+storiesOf('UpdaterPage', module).
+  add('Normal', () => (
+    <UpdaterPage
+      appName={appName}
+      notifyOnly={false}
+      isDownloading={false}
+      progress={0}
+      onClickInstall={action('clicked install')}
```

Comment:
Why doesn't this match anything? Is it supposed to be `click-install`?

Reply:
Yes. On the actual app, `click-install` ipc message is sent from the updater dialog to the main process in order to invoke the function of electron-updater.